### ISSUE TITLE
Nice work, can you kindly attach a screenshot of the final work?

### ISSUE THREAD

**Issue by yidea** [Frijol/cat-feeder#1]
Nice work, can you kindly attach a screenshot of the final work?

**Comment by Frijol** [comment]
Good point! I didn't finish writing this up! Mostly because I'm ashamed of the mechanical linkage I cobbled together. But I'll try to update the readme in the next week.
